aospy/automate.py

```diff
@@ -290,34 +290,126 @@ def _print_suite_summary(calc_suite_specs):
+              - 'ts' : Gridpoint-by-gridpoint time-series
+
+            - Averages over each region specified via `region`:
+
+              - 'reg.av', 'reg.std', 'reg.ts' : analogous to 'av', 'std', 'ts'
+
+        output_vertical_reductions : {None, 'vert_av', 'vert_int'}, optional
+            How to reduce the data vertically:
+
+            - None : no vertical reduction
+            - 'vert_av' : mass-weighted vertical average
+            - 'vert_int' : mass-weighted vertical integral
+        input_time_intervals : {'annual', 'monthly', 'daily', '6hr', '3hr'}
+            A string specifying the time resolution of the input data.
+        input_time_datatypes : {'inst', 'ts', 'av'}
+            What the time axis of the input data represents:
+
+            - 'inst' : Timeseries of instantaneous values
+            - 'ts' : Timeseries of averages over the period of each time-index
+            - 'av' : A single value averaged over a date range
+
+        input_vertical_datatypes : {False, 'pressure', 'sigma'}, optional
+            The vertical coordinate system used by the input data:
+
+            - False : not defined vertically
+            - 'pressure' : pressure coordinates
+            - 'sigma' : hybrid sigma-pressure coordinates
+
+        input_time_offsets : {None, dict}, optional
+            How to offset input data in time to correct for metadata errors
+
+            - None : no time offset applied
+            - dict : e.g. ``{'hours': -3}`` to offset times by -3 hours
+              See :py:meth:`aospy.utils.times.apply_time_offset`.
+
     exec_options : dict or None (default None)
         Options regarding how the calculations are reported, submitted, and
         saved.  If None, default settings are used for all options.  Currently
         supported options (each should be either `True` or `False`):
 
-        - prompt_verify : If True, print summary of calculations to be
-              performed and prompt user to confirm before submitting for
-              execution
-        - parallelize : If True, submit calculations in parallel
-        - write_to_tar : If True, write results of calculations to .tar files,
-              one for each object.  These tar files have an identical directory
-              structures the standard output relative to their root directory,
-              which is specified via the `tar_direc_out` argument of each Proj
-              object's instantiation.
+        - prompt_verify : (default False) If True, print summary of
+              calculations to be performed and prompt user to confirm before
+              submitting for execution.
+        - parallelize : (default False) If True, submit calculations in
+              parallel.
+        - write_to_tar : (default True) If True, write results of calculations
+              to .tar files, one for each object.  These tar files have an
+              identical directory structures the standard output relative to
```

Comment:
Good catch!  I agree, this is a bug in my view.  I will fix it within this PR if you agree.